sdk/aqueduct/aqueduct_client.py

```diff
@@ -334,10 +334,11 @@ def publish_flow(
             retention_policy=retention_policy,
         )
 
-        if self._in_notebook_or_console_context:
-            _show_dag(self._api_client, dag)
-
         flow_id = self._api_client.register_workflow(dag).id
+
+        url = "http://" + self._api_client.aqueduct_address + "/workflow/" + str(flow_id)
```

Comment:
we shouldn't assume that this is http. Lets expose a method on the APIClient that returns the address prefix? (`http(s)://<address>)